library/core/src/num/nonzero.rs

```diff
@@ -287,6 +287,7 @@ impl<T> From<NonZero<T>> for T
 where
     T: ZeroablePrimitive,
 {
+    /// Get the inner num of `NonZero`.
```

Comment:
Maybe look a the docs of `get` and repeat them here, or just state that it's equivalent to `get` and link to it